ratings/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Define RatingView style using all the attributes under VitaminRatingView -->
+    <attr name="vtmnRatingViewStyle" format="reference" />
+    <!-- Define the style of the RatingView's ratingBar -->
+    <attr name="vtmnRatingViewRatingBarStyle" format="reference"/>
```

Comment:
If you accept the renaming of the style, this should be edited to `vtmnRatingViewBarStyle`

Reply:
Changed